package/boot/uboot-envtools/files/ramips

```diff
@@ -31,6 +31,7 @@ wsr-600|\
 zbt-wg2626)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x10000"
 	;;
+linksys,ea8100|\
```

Comment:
Please make this separate commit `uboot-envtools: add support for Linksys EA8100`